docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '2'
+
+services:
+    nginx:
+        image: nginx:latest
+        container_name: cqrs_codely_tv_nginx
+        ports:
+            - "80:80"
+        volumes:
+            - .:/app
+            - ./etc/infrastructure/nginx/site.conf:/etc/nginx/conf.d/default.conf
+        depends_on:
+            - php
+        volumes_from:
+            - php
+        links:
```

Comment:
[The links option is already deprecated](https://docs.docker.com/compose/compose-file/compose-file-v2/#links). In fact, since you're already specifying the `depends_on` attribute, we don't need to add the `links` one in order to have visibility between these two containers. [Here you have more information on that](https://stackoverflow.com/a/39658359/967124).

Reply:
Removed the links and volumes_froms